Sources/SwiftRedis/Redis.swift

```diff
@@ -48,9 +48,15 @@ public class Redis {
             callback(nil)
         }
     }
+
+    /// Disconnects from a redis server
+    public func disconnect(){
+         respHandle?.disconnect()
+     }
     
     /// Authenticate against the server
     ///
+    /// - Parameter username: String for the username (Redis 6+ only).
```

Comment:
Could you document the new function with all the fields, and not that field (`username`) that doesn't belong to this function?